plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/command/RemoveConnectionHandler.java

```diff
@@ -41,7 +44,7 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
 
 									@Override
 									public String toString(IConnection connection) {
-										return "\n" + connection.toString();
+										return connection == null ? "\nnull" : ("\n" + connection.toString());
```

Comment:
there's no point in removing a null connection.
The confirmation dialog should be opened only if the list of connections contains valid values.